docs/en/terminology_server/on_premise.md

```diff
@@ -62,3 +62,7 @@ curl -sSL https://s3.us-east-1.amazonaws.com/artifacts.terminologyservice.johnsn
     </td>
   </tr>
 </table>
+
+## API Usage
+
```

Comment:
Can you add some filler words like,
Once you have deployed Terminology Server to your premise, you have API access to it. For Reference, see this